### PR TITLE
fix typo in probe.go

### PR DESCRIPTION
**Pull request by yutachaos**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

> /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/probe/probe.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package probe
 
-// Result is a string used to handle the results for probing container readiness/livenss
+// Result is a string used to handle the results for probing container readiness/liveness
```
Comment (draveness) [inline pkg/probe/probe.go]: /priority backlog
/ok-to-test
/kind cleanup
/lgtm

**Comment by yutachaos** [comment]
I regsted CLA and fixed commit user.email

**Comment by liggitt** [comment]
this is in the merge pool and will merge once master reopens for 1.16

/unassign

**Pull request merged**